src/lib/y2storage/planned/bcache.rb

```diff
@@ -0,0 +1,78 @@
+# encoding: utf-8
+
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "y2storage/planned/device"
+require "y2storage/planned/mixins"
+require "y2storage/match_volume_spec"
+
+module Y2Storage
+  module Planned
+    # Specification for a Y2Storage::Bcache object to be created during the proposal
+    #
+    # @see Device
+    class Bcache < Device
+      include Planned::CanBeFormatted
+      include Planned::CanBeMounted
+      include Planned::CanBeEncrypted
+      include MatchVolumeSpec
+
+      # @return [String] tentative device name of the bcache
```

Comment:
BTW, is clearly stated in the corresponding AY documentation that the name used in the profile for bcache devices is just to identify them in the profile itself and that users shoudn't expect those names to be final?

Reply:
No, I will add a note.

Reply:
Now it is :)